mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/AnnotationMapperTest.java

```diff
@@ -68,6 +69,8 @@ public final void setProducer(IProducer<T> producer) throws MailboxException {
         this.producer = producer;
         this.annotationMapper = newInstance.createAnnotationMapper();
         this.mailboxId = newInstance.generateId();
+
+        Assume.assumeNotNull(annotationMapper);
```

Comment:
Capabilities are defined at the mailbox manager level, not at the mapper level.

And we can imagine (what is the case for JPA, see below message) having some capabilities but being not able to handle MapperTests (because IE you don't have AttachmentMapper)